src/main/java/com/device/shop/controller/OrderItemsController.java

```diff
@@ -0,0 +1,45 @@
+package com.device.shop.controller;
+
+import com.device.shop.entity.OrderItems;
+import com.device.shop.exception.BadRequestException;
+import com.device.shop.service.OrderItemsService;
+
+import lombok.AllArgsConstructor;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@AllArgsConstructor
+public class OrderItemsController {
+
+    OrderItemsService orderItemsService;
+
+    @GetMapping("/order/{id}/items")
+    public ResponseEntity<OrderItems> getOrderItemsId(@PathVariable("id") Long orderItemsId) {
+        OrderItems orderItems = orderItemsService.getOrderItemsById(orderItemsId);
+        return new ResponseEntity<>(orderItems, HttpStatus.OK);
+    }
+
+    @PostMapping("/order/items/{id}")
```

Comment:
Change `id` to `productId` in the path because it is not clear that this is productId